PR - Z-Wave Window Shade

habhomegit opened the pull request:
PR for Z-Wave Window Shade.  Adding reverse function, Switch and Switchlevel capabilities.

iblinds committed:
PR - Z-Wave Window Shade

PR for Z-Wave Window Shade.  Adding reverse function, Switch and Switchlevel capabilities.

dkirker committed:
Rename DTH

dkirker committed:
Add Window Shade Level

dkirker committed:
Code cleanup

dkirker committed:
More cleanup

dkirker committed:
More cleanup

dkirker committed:
Add beginning of logic organization for handling Z-Wave functional device use cases

dkirker commented:
I'm pushing up some of the code that I have. It isn't quite usable yet, but best to have it backed up. :)

dkirker committed:
Spacing and notes with logic that is being filled in

dkirker committed:
Update `setShadeLevel` to use new logic

dkirker committed:
Add main logic to convert between capability and device levels

dkirker committed:
Fix exception

dkirker committed:
Update code to toy with the idea of directly controlling the device's native level using Switch Level
Until there is a way to properly express the slat direction (for use in rules, SmartApps, and preferably by voice) this allows the user the ability to do this (without needing to set and unset the reverse preference, which can't be done programatically).

iblinds committed:
Removing version V3 fingerprint, publishing a new Device Handler for V3

habhomegit committed:
Removing V3 fingerprint

Creating a new V3 Device Handler

habhomegit committed:
Merge pull request #1 from habhomegit/habhomegit-patch-1

Removing V3 fingerprint
